fw/main.c

```diff
@@ -409,6 +471,14 @@ static struct ctl_table tfw_sysctl_tbl[] = {
 		.mode		= 0644,
 		.proc_handler	= tfw_ctlfn_state_io,
 	},
+#ifdef DBG_ERRINJ
+	{
+		.procname	= "errinj",
```

Comment:
In short, there is some runtime API and some C API and we do not have any examples of the usage at the moment. Could you please add an example and a short HowTo to the wiki? Maybe extend https://github.com/tempesta-tech/tempesta/wiki/Debugging-and-troubleshooting or create a new page under Contributing.

Reply:
Fixed